### PR TITLE
bugfix: The color characters are messy when running by 'seata-server.bat'

### LINKED ISSUE #2954: seata 1.3.0 使用 seata-server.bat windows 启动乱码问题
## Why you need it?
 Is your feature request related to a problem? Please describe in details

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

### PR DESCRIPTION
**Pull request by wangliang181230**
### Ⅰ. Describe what this PR did
bugfix: The color characters are messy when running by `seata-server.bat`.
BUG修复：当使用`seata-server.bat`运行时，颜色符号变成乱码的问题修复。

### Ⅱ. Does this pull request fix one issue?

fixes #2954

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -205,6 +205,8 @@
                                     <extraArgument>-verbose:gc</extraArgument>
                                     <!--netty-->
                                     <extraArgument>-Dio.netty.leakDetectionLevel=advanced</extraArgument>
+                                    <!--run by *.bat or *.sh-->
+                                    <extraArgument>-Drun.by-startup=true</extraArgument>
```
Comment (slievrly) [inline server/pom.xml:209]: wins and unix both contain `-Drun.by-startup=true`. How distinguish between them?

File: server/src/main/java/io/seata/server/env/ContainerHelper.java [hunk 2]
```diff
@@ -59,6 +61,20 @@ public static boolean isRunningInContainer() {
...
+     *
+     * @return If application is run in windows with seata-server.bat.
+     */
+    public static boolean isRunningInWindowsWithBat() {
+        if (!PlatformDependent.isWindows()) {
+            return false;
+        }
+
+        String runByStartup = System.getProperty("run.by-startup");
+        return runByStartup != null && runByStartup.equalsIgnoreCase("true");
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/env/ContainerHelper.java:75]: Boolean.parseBoolean

Comment (wangliang181230) [reply server/src/main/java/io/seata/server/env/ContainerHelper.java]: done

File: server/pom.xml (hunk 1 above)
Comment (wangliang181230) [reply server/pom.xml]: I renamed the jvm argument name. PTAL.

**Review by slievrly — APPROVED** [review]
LGTM

**Review by a364176773 — APPROVED** [review]
LGTM

**Pull request merged**